elements/pfe-band/README.md

```diff
@@ -0,0 +1,32 @@
+# PatternFly Element | Band Element
```

Comment:
Can we get more detail here? Show usage, attributes, slots, and any custom events that are fired.

Reply:
Also, any exposed CSS variables should be mentioned here.

Reply:
+1 this is now done